tbx/services/blocks.py

```diff
@@ -38,8 +40,44 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/partners_block.html"
 
 
-class ServiceStoryBlock(StoryBlock):
+class CaseStudyStructValue(blocks.StructValue):
+    @cached_property
+    def featured_case_study_image(self):
+        if image := self.get("image"):
+            return image
+        elif page := self.get("link"):
+            model = page.content_type.model_class().__name__
+            if model == "WorkPage":
+                return page.specific.header_image
+            elif model == "HistoricalWorkPage":
+                return page.specific.feed_image
+        return None
+
+    @cached_property
+    def featured_case_study_logo(self):
+        if logo := self.get("logo"):
+            return logo
+        elif page := self.get("link"):
+            if page.content_type.model_class().__name__ == "WorkPage":
+                return page.specific.logo
+        return None
+
+
+class FeaturedCaseStudyBlock(blocks.StructBlock):
+    link = blocks.PageChooserBlock(
```

Comment:
I noticed when selecting a page that the new work pages come after all the historical ones - I don't know if there is any way to change the ordering to get the new work pages first?

Reply:
from https://docs.wagtail.org/en/v5.2.3/reference/streamfield/blocks.html#wagtail.blocks.PageChooserBlock it seems there are no options available to control the ordering of the pages. Achieving this would most likely require some bit of customisation, and may be related to https://github.com/wagtail/wagtail/issues/882